### PR TITLE
Escape characters where appropriate

### PR DESCRIPTION
**Pull request by aroc**
## What does this PR do?
It adds escaping of characters in the appropriate places before presenting the values in the visualizations and visualization wrapper.

## How can I test this?
1. Ensure all tests pass.
2. Try adding `<script>alert('hi');</script>` in data values and ensure they are properly escaped for all chart types.

### CODE REVIEW

**Comment by dustinlarimer** [comment]
Tests are passing :+1: reviewing now-

screen shot 2016-08-11 at 4 28 46 pm

**Comment by dustinlarimer** [comment]
Looks good! Nice work 👍

File: lib/utils/escape-html.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        break;
+      case 39: // '
+        escape = '&#39;';
+        break;
+      case 60: // <
+        escape = '&lt;';
+        break;
+      case 62: // >
+        escape = '&gt;';
+        break;
```
Comment (eliot-keen) [inline lib/utils/escape-html.js]: Shouldn't you escape ' to &#x27 and / to &#x2F as well?

File: lib/index.js [hunk 2]
```diff
@@ -239,20 +239,20 @@
     if (this.data()[0].length === 2 && !isDateString(this.data()[1][0])) {
       this.dataset.updateColumn(0, function(value){
```
Comment (eliot-keen) [inline lib/index.js]: Into what HTML elements are these escaped values inserted?

**Pull request merged**